src/schema/posts.ts

```diff
@@ -621,6 +658,31 @@ export const typeDefs = /* GraphQL */ `
   }
 `;
 
+const saveMentions = (
```

Comment:
I think we have a similar function for the comments, can't we generalize with function params perhaps?